tutor/templates/build/openedx/settings/partials/assets.py

```diff
@@ -5,9 +5,6 @@
 from openedx.core.lib.derived import derive_settings
 
 ENABLE_COMPREHENSIVE_THEMING = True
-COMPREHENSIVE_THEME_DIRS.append('/openedx/themes')
```

Comment:
On my system, `ENABLE_COMPREHENSIVE_THEMING` is already enabled and have confirmed this using shell of lms container. and in ENV variables, `COMPREHENSIVE_THEME_DIRS` is added. the problem is LMS doesn't pick `COMPREHENSIVE_THEME_DIRS` from `Env` variables. It picks its value using `themes_dirs = get_theme_base_dirs_from_settings(settings.COMPREHENSIVE_THEME_DIRS)`. 

There are two options here:
1. move `COMPREHENSIVE_THEME_DIRS` back to settings variable
2. Make changes in edx-platform to pick `COMPREHENSIVE_THEME_DIRS` value using `get_env_setting`

Any of these solution is not being done in https://github.com/openedx/edx-platform/pull/34318/files.

<img width="832" alt="image" src="https://github.com/overhangio/tutor/assets/112946189/27e399db-c08a-4e8d-9c5e-494b71fd54f9">


If there seems other alternative solution, feel free to propose here.

Reply:
Thank you for testing @hinakhadim . Are you sure you are on the latest master for edx-platform? It's possible I missed something, but my intention was to load `settings.COMPREHENSIVE_THEME_DIRS` from the environment variable of the same name:  https://github.com/openedx/edx-platform/blob/0d4adaa5d70364fe9ff0857b15cbc58cf952f584/cms/envs/common.py#L2205

> ```
> COMPREHENSIVE_THEME_DIRS = os.environ.get("COMPREHENSIVE_THEME_DIRS", "").split(":")
> ```

This behavior was added in this commit https://github.com/openedx/edx-platform/commit/21a1235a284d9aa4891dc75b31777df31255fc62 ([original PR](https://github.com/openedx/edx-platform/pull/34554))

Reply:
Thanks for information, @kdmccormick. I have tested with latest master for edx-platform and it is now loading the indigo theme and picking value of `COMPREHENSIVE_THEME_DIRS` from `env`. So, everything is fine from my side.